### ISSUE TITLE
JQL - new CommitId filter

### ISSUE THREAD

**Issue by bartoszwalacik** [javers/javers#309]
Requested by @atomfrede
https://github.com/deepu105/generator-jhipster-entity-audit/issues/4

Query by CommitId could be implemended as a new filter, available only for snapshot queries:

```
javers.findSnapshots( QueryBuilder.byInstanceId("bob", Employee.class).withCommitId(1.0).build() )
```

**Comment by atomfrede** [comment]
I can try to contribute the new filter if you like.

**Comment by bartoszwalacik** [comment]
We have a team meeting today evening, and we will do some planning. Stay tuned

**Comment by atomfrede** [comment]
Awesome! Thanks for your support!

**Comment by bartoszwalacik** [comment]
@atomfrede, @mwesolowski is going to implement this new filter. Stay tuned

**Comment by bartoszwalacik** [comment]
released in JaVers v 1.4.5
see http://javers.org/documentation/jql-examples/#commit-id-filter

**Comment by bartoszwalacik** [comment]
@atomfrede check new filter by commitId(). This filter makes sense only for snapshot queries
